server.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable func-names */
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable import/newline-after-import */
+/* eslint-disable linebreak-style */
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] **Optional:** Kindly put some effort to solve the linters. The linter rules are in place to enforce uniform code guidelines in the community.